### PR TITLE
add simnet tests

### PR DESCRIPTION
**Pull request by radupopa2010**
This PR adds directory which will hold high level integration tests that will run with the simnet set of tools.

At the moment those kind of tests are maintained by the simnet team in simnet repo which is hosted just on gitlab.
This was done in this manner while we developed working version of tools to spawn a simulation network based on a docker image and run tests against this network.

There will be a follow up PR to edit the Ci file of this project, but I need those files hosted in this repo before.

We are still developing simnet and we are thinking how to make it simpler to use.

By moving simnet tests into this repository, the expectations are that developers would be able to maintain and create new tests.
Off course simnet team will help explaining how things work and how to use simnet tools

### CODE REVIEW

**Comment by drahnr** [comment]
Generally looks good, it would be nice to add a link to the introduction presentation video of `gurke`/`simnet`.

**Review by drahnr — APPROVED** [review]

**Comment by grbIzl** [comment]
Please remove simnet_tests/run_huge_network.sh file, it's not intended for CI now

File: simnet_tests/README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+Behavior-Driven Development test files that describe tests in plain English.
+Under the hood there are assertions that specific metrics should have specific
+values. 
+
+At the moment we have only one test for parachains: `/parachains.features`
+This test uses a JS script that we added to simnet image and it's launched
+by this step in the cucumber file:
+` Then launch 'node' with parameters '--unhandled-rejections=strict /usr/local/bin/simnet_scripts test_parachain ./configs/adder.json ws://localhost:11222 100 10'`
+
+`run_test.sh` is a script meant to ease up launching a test.
```
Comment (grbIzl) [inline simnet_tests/README.md:25]: I suggest the following wording for run_test.sh: is an entry point for running all tests in the folder. Any setup required for tests (but cannot be done in configs) is performed here. The main script's responsibility is to run gurke (link to gurke repo) with passed parameters.

File: simnet_tests/README.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+# Simulation tests, or high level integration tests.
```
Comment (grbIzl) [inline simnet_tests/README.md:1]: Also an idea: should we describe the content of simnet image with links to the corresponding repos somewhere? (We can do that in gitlab and provide here and in substrate copy the link to the description).

Comment (radupopa2010) [reply simnet_tests/README.md:1]: yeah, I wanted to do that also. I waited for an answer from Ben, but then I forgot to add the description and links

**Review by grbIzl — APPROVED** [review]

**Comment by radupopa2010** [comment]
> Generally looks good, it would be nice to add a link to the introduction presentation video of `gurke`/`simnet`.

I added the link to barcap presentation. Good suggestion.
